worker/include/ObjectPoolAllocator.hpp

```diff
@@ -0,0 +1,89 @@
+#ifndef MS_OBJECT_POOL_ALLOCATOR_HPP
+#define MS_OBJECT_POOL_ALLOCATOR_HPP
+
+// #define MS_ALLOCATOR_FREE_ON_RETURN 1
```

Comment:
Is this ready or not? Shouldn't this be uncommented and the condition below removed and always do this instead?

Reply:
👍